packages/@o3r/rules-engine/src/engine/operator/operators/basic.operators.ts

```diff
@@ -1,14 +1,21 @@
-import { isString, isSupportedSimpleTypes, parseRegExp } from '../operator.helpers';
-import { Operator, SupportedSimpleTypes, UnaryOperator } from '../operator.interface';
+import {
+  isString,
+  isSupportedSimpleTypes,
+  parseRegExp
+} from '../operator.helpers';
+import {
+  Operator,
+  SupportedSimpleTypes,
+  UnaryOperator
+} from '../operator.interface';
 
 /**
  * Check if a variable is equal to a specific value
  * @title is equal to
  */
 export const equals: Operator = {
   name: 'equals',
-  // eslint-disable-next-line
-  evaluator: (firstValue, secondValue) => firstValue == secondValue
+  evaluator: (firstValue, secondValue) => firstValue === secondValue
```

Comment:
this no totally equivalent
`equals(4, '4')` was previously true, it is now false

Reply:
Especially when the value might come from a JSON file (if one of the value is a const defined on a CMS)

Reply:
And this is true for the whole file :)